# Generated for NaoQi method ALMemory.unsubscribeToEvent
# Get event list generated by extractor

# Event name
string name
# subscriber name
string moduleName
---


